test_app/migrations/0002_set_up_resources_test_data.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 4.2.8 on 2024-01-26 11:03
+
+from django.db import migrations, models
+
+
+def create_test_data(apps, schema_editor):
+    """
+    This migration sets up some resource instances after the resource registry is
+    created. This is to test two things:
+        1. That the post_save and post_delete signals in the resources app do not
+           fire during migrations.
+        2. That the post migration signal to initialize resource instances from
+           existing objects works correctly.
+    """
+    ResourceMigrationTestModel = apps.get_model("test_app", "ResourceMigrationTestModel")
+    Resource = apps.get_model("dab_resource_registry", "Resource")
+
+    ResourceMigrationTestModel.objects.create(name="migration resource")
+    r2 = ResourceMigrationTestModel.objects.create(name="migration resource 2")
+
+    assert Resource.objects.all().count() == 0
```

Comment:
You could do this with an existing model like Organization. Having assertions here is otherwise fine by me.

Reply:
I did this with orgs first, but it broke other tests that assume they are getting a blank slate, so I figured that creating a new model specifically for this test was the best course of action.